src/gallery/tracker/qgallerytrackerresultset.cpp

```diff
@@ -371,7 +378,11 @@ void QGalleryTrackerResultSetPrivate::syncFinish(const int rIndex, const int iIn
 
 bool QGalleryTrackerResultSetPrivate::waitForSyncFinish(int msecs)
 {
+#if QT_VERSION >= QT_VERSION_CHECK(6, 0, 0)
+    QTimer timer;
```

Comment:
I think QTimer should work for all Qt versions that matter. Let's just switch to that? Applies on many places.